test-server/app.py

```diff
@@ -1,6 +1,18 @@
 
+import os
+
+
+if os.environ.get('NEW_RELIC_LICENSE_KEY'):
+    try:
+        import newrelic.agent
+        newrelic.agent.initialize(os.path.abspath(os.path.join(os.path.dirname(__file__), 'newrelic.ini')))
+    except ImportError:
+        pass
+
```

Comment:
a warning log message would be nice here: if somebody set the license key, she will probably expect it to be activated